Remove the need to run docker-auto-labels

m1yag1 opened the pull request.

therealmarv reviewed (changes requested):
Remove docker-auto-labels instructions fully from https://github.com/openstax/output-producer-service/blob/master/docs/operations/updating_the_stack.rst too?

Are the new instructions on how to create labels missing?

m1yag1 commented:
The instructions on how to create the labels are there in the changes.

m1yag1 commented on docs/operations/setting_up_the_swarm.rst:
```diff
@@ -161,13 +161,15 @@ Create Main Traefik Service
 .. code-block:: bash
 
    docker node update --label-add proxy=true $NODE_ID
+   docker node update --label-add app-db-data=true $NODE_ID
```
@therealmarv this adds the app-db-data=true label to the first node in the swarm.

m1yag1 commented on docker-compose.deploy.volumes-placement.yml:
```diff
@@ -7,7 +7,7 @@ services:
         constraints:
           - node.role == manager
-          - node.labels.${STACK_NAME}.app-db-data == true
+          - node.labels.app-db-data == true
```
This sets the label for the database volume to the one set in the documentation. I've already done this in the swarm.

m1yag1 commented:
@therealmarv I believe I resolved the comments. I removed mention of docker-auto-labels in the docs and I also removed the documentation on creating the tunnel as that is not needed when paramiko is also installed. I've tested this out with the new CORGI CD pipeline and works great. We're able to set the ssh host and docker takes care of the rest.

therealmarv reviewed (approved):
Looks good now for me.
